src/main/java/uk/gov/hmcts/reform/civil/service/GeneralAppFeesService.java

```diff
@@ -24,19 +26,30 @@
 public class GeneralAppFeesService {
 
     private static final BigDecimal PENCE_PER_POUND = BigDecimal.valueOf(100);
-
+    private static final int FREE_GA_DAYS = 14;
     private final RestTemplate restTemplate;
     private final GeneralAppFeesConfiguration feesConfiguration;
 
     public Fee getFeeForGA(CaseData caseData) {
         String queryURL = feesConfiguration.getUrl() + feesConfiguration.getEndpoint();
+        String keyword = getKeyword(caseData);
         UriComponentsBuilder builder = UriComponentsBuilder.fromUriString(queryURL)
                 .queryParam("channel", feesConfiguration.getChannel())
                 .queryParam("event", feesConfiguration.getEvent())
                 .queryParam("jurisdiction1", feesConfiguration.getJurisdiction1())
                 .queryParam("jurisdiction2", feesConfiguration.getJurisdiction2())
                 .queryParam("service", feesConfiguration.getService())
-                .queryParam("keyword", getKeyword(caseData));
+                .queryParam("keyword", keyword);
+        //TODO remove this if block after we have real free fee for GA
```

Comment:
because now we have no free fee data. so we borrowed one from insolvency service. once payment team give us the new free fee code for GA, we will delete the the whole if block